src/WireMock/Client/XPathValueMatchingStrategy.php

```diff
@@ -4,13 +4,7 @@
 
 class XPathValueMatchingStrategy extends ValueMatchingStrategy
 {
-    /**
-     * @var string|AdvancedPathPattern
-     * @serde-named-by matchingType
-     * @serde-possible-names matchingValueNames
-     */
-    protected $matchingValue;
```

Comment:
the property is defined in the ValueMatchingStrategy, can I delete them without breaking Serde?